CHANGELOG.md

```diff
@@ -108,6 +108,8 @@
   [#9337](https://github.com/Kong/kong/pull/9337)
 - Fixed an issue with error-handling and process cleanup in `kong start`.
   [#9337](https://github.com/Kong/kong/pull/9337)
+- Now data planes' node IDs persist after restarting.
```

Comment:
Since the current implementation is expected to use random IDs, this PR is a new feature.